README.md

```diff
@@ -2,6 +2,8 @@
 
 S3 as an ObjectStore for [Datafusion](https://github.com/apache/arrow-datafusion).
 
+Initial implementation donated by [Mike Seddon](https://github.com/seddonm1).
```

Comment:
I dont think you really need to add this. If you really want to then it can be hidden down the bottom to make it less prominent.

Reply:
kk, will remove.